example-react-app/src/dev/palette.tsx

```diff
@@ -135,9 +135,13 @@ import {
   Variant,
   Palette
 } from "@haulmont/react-ide-toolbox";
+import { Field } from "@haulmont/jmix-react-antd";
 import { FormattedMessage } from "react-intl";
 import * as React from "react";
 
+const ENTITY_NAME = "";
+const readOnlyMode = false;
```

Comment:
Why can't we inline these values?

Reply:
We need to preserve naming of these values so dragged from palette element could use scoped screen variables

Reply:
@web-devel Is this the right way to approach this? What if multiple components in the palette wants to use same scoped variables?